source/agora/consensus/Fee.d

```diff
@@ -435,29 +435,31 @@ public class FeeManager
 
     /***************************************************************************
 
-        Calculate the transaction fee and adjust the fee based on the
-        transaction's size measured in bytes.
+        Calculate the transaction fee rate.
 
-        The bigger the transaction size is, the smaller the the fee becomes.
+        This calculates the fee rate of the transaction, and can be used to
+        compare two transactions for inclusion in a block / pool eviction.
+        If the fee is not an exact multiplicator of the tx size,
+        the remainder will be ignored.
 
         Params:
-            tx = transaction for which we want to calculate the adjusted fee
-            peekUTXO = UTXO finder
-            tot_fee = adjusted fee
+            tx = transaction for which we want to calculate the fee rate
+            peekUTXO = UTXO finder (with or without replay protection)
+            rate = This transaction minimum fee rate
 
         Returns: string describing the error, if an error happened, null otherwise
 
     ***************************************************************************/
 
-    public string getAdjustedTXFee (in Transaction tx, scope UTXOFinder peekUTXO,
-        out Amount tot_fee) nothrow @safe
+    public string getTxFeeRate (in Transaction tx, scope UTXOFinder peekUTXO,
+        out Amount rate) nothrow @safe
     {
         try
-            tot_fee = tx.getFee(peekUTXO);
+            rate = tx.getFee(peekUTXO);
```

Comment:
Just a comment.
At first I thought there was an error in this statement. But it is correct as we need to create the `Amount` before we do the division. 
I wish we returned the result in `Amount` calculations so we could avoid confusing code statements like this one. Not part of this PR but I think we could look at making `Amount` easier to work with.

Reply:
Yep, had the same thought. Will add a comment.